modules/Behavior.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('underscore');
+var TorsoCell = require('./Cell');
+
+// Map of eventName: lifecycleMethod
+var eventMap = {
+  'before-attach': '_attached',
+  'before-detach':  '_detached',
+  'before-activate': '_activate',
+  'before-deactivate': '_deactivate',
+  'before-dispose': '_dispose',
+  'render:before-attach-tracked-views': 'attachTrackedViews',
+  'render:begin': 'prerender',
+  'render:complete': 'postrender',
+  'initialize:begin':  '_preinitialize',
+  'initialize:complete': '_postinitialize'
+};
+
+module.exports = TorsoCell.extend({
+  constructor: function(options) {
+    options = options || {};
+    this.view = options.view;
+    this._bindLifecycleMethods();
+    this._bindEventCallbacks();
+    TorsoCell.apply(this, arguments);
+  },
+
+  _bindLifecycleMethods: function() {
+    _.each(eventMap, function(callback, event) {
+      this.listenTo(this.view, event, this[callback]);
+    }, this);
+  },
+
+  _bindEventCallbacks: function() {
+    var behaviorEvents = _.result(this, 'events');
+    var viewEvents = this.view && this.view.events;
+
+    if (!viewEvents) {
+      if (!behaviorEvents) {
+        return;
+      } else {
+        viewEvents = {};
+      }
+    }
+
+    var boundBehaviorEvents = this._bindEventCallbacksToBehavior(behaviorEvents);
+
+    if (_.isFunction(viewEvents)) {
```

Comment:
A bit. In View.bindEvents, the View will attempt to bind the behavior event callbacks to the view's context, although when you bind a method multiple times, the first bind wins. The logic in _bindEventCallbacksToBehavior to find the method is also duplicated. I don't see a way of sharing the logic without modifying Backbone.